lib/mangadex/statistic.rb

```diff
@@ -17,5 +17,17 @@ def self.get(uuid)
         '/statistics/manga/%{uuid}' % {uuid: uuid},
       )
     end
+
+    sig { params(args: T::Api::Arguments).returns(T::Api::GenericResponse) }
+    def self.list(**args)
+      to_a = Mangadex::Internal::Definition.converts(:to_a)
+
+      Mangadex::Internal::Request.get(
+        '/statistics/manga',
+        Mangadex::Internal::Definition.validate(args, {
```

Comment:
This API used for validating params and making sure the right request is made is slightly confusing, I may work on improving it in the future

Reply:
Sweet. So that's how you do it. Were you ever able to get it from their https://api.mangadex.org/docs/swagger.html#/ ? They didn't seem to accept multiple string uuids (I always got a 400).

Reply:
Ohhh interesting, I use this one:

<img width="388" alt="image" src="https://user-images.githubusercontent.com/14348784/199290246-d0150c80-5907-4f12-bb86-fad782a56c1e.png">

The change I made above calls `.to_query` on the param ([ref](https://github.com//thedrummeraki/mangadex/blob/a067406317eb6dd4c500c5644eec9016649d3632/lib/extensions.rb#L4)), which ultimately generates the right URL query params for the everything:

```ruby
Mangadex::Statistic.list(manga: ['uuid1', 'uuid2'])
```

should generate

```
https://api.mangadex.org/statistics/manga?manga[]=uuid1&manga[]=uuid2
```

Did you get a URL similar to this when writing #38?

Reply:
<img width="1485" alt="Screen Shot 2022-11-01 at 1 40 23 PM" src="https://user-images.githubusercontent.com/845626/199301038-e5373f01-9aa6-45f4-a120-85f1b3677294.png">

Yeah, I used the same one. It was probably syntax, if you try it on swagger (as per the pic), you'd get a 400.

Test uuid: **7dbc59db-6c77-4084-94ec-a38e0826faab** and **ce7cd7b0-e595-4f0d-98e8-447da10d652d**

Reply:
Ohh ok now I understand, yeah it's super weird that swagger is broken like this 🤔 `manga[]=7dbc59db-6c77-4084-94ec-a38e0826faab,ce7cd7b0-e595-4f0d-98e8-447da10d652d` will definitely not work since it'll mostly likely be read as a String of `"7dbc59db-6c77-4084-94ec-a38e0826faab,ce7cd7b0-e595-4f0d-98e8-447da10d652d"` instead of an Array on Mangadex side, ohh well 🤷